packages/libncurses/libncurses.spec

```diff
@@ -1,5 +1,5 @@
-%global ncurses_ver 6.4
-%global ncurses_rev 20231209
+%global ncurses_ver 6.5
+%global ncurses_rev 20240504
```

Comment:
Why this version and not a later version?

Reply:
Another update I missed. Testing the latest one now